specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json

```diff
@@ -16,19 +16,22 @@
     "application/json"
   ],
   "paths": {
-    "/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/cancel": {
-      "post": {
-        "operationId": "Subscriptions_Cancel",
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/OperationStatus/{Id}": {
```

Comment:
I'm not sure whether you need to model this - does it make sense for customers to be able to call it directly, or only within the context of an async operatin? The generated SDKs will be able to track async operations without adding this definition.